src/cs-utils/convert.hpp

```diff
@@ -79,16 +79,51 @@ CS_UTILS_EXPORT glm::dvec3 lngLatToNormal(glm::dvec2 const& lngLat, double radiu
 /// Returns the geodetic coordinates (lng, lat) for a given normal vector.
 CS_UTILS_EXPORT glm::dvec2 normalToLngLat(glm::dvec3 const& normal, double radiusE, double radiusP);
 
-/// Convert boost::posix_time::ptime to spice time, which is defined by the
-/// Barycentric Dynamical Time.
-CS_UTILS_EXPORT double toSpiceTime(boost::posix_time::ptime const& tIn);
-
-/// Convert a time string to spice time, which is defined by the Barycentric Dynamical Time.
-CS_UTILS_EXPORT double toSpiceTime(std::string const& tIn);
-
-/// Convert time in seconds since 2000-01-01 12:00:00.000 to boost::posix_time::ptime. Be
-/// aware, that fractional seconds will be truncated. //DocTODO
-CS_UTILS_EXPORT boost::posix_time::ptime toBoostTime(double tIn);
+/// Time in CosmoScout VR is passed around in different formats.
+/// * Strings usually store time in the ISO format YYYY-MM-DDTHH:MM:SS.fffZ. The 'Z' suffix is not
+///   really required on the C++ side, as time strings are always considered to be in UTC. This
+///   format is also directly convertible to JavaScript Dates, here however the 'Z' is required!
+///   Else the Date object will be in your local time zone. So it's a good practive to always append
+///   the 'Z'.
+/// * boost::posix_time::ptime is used for conversions and is also always in UTC.
+/// * SPICE time is stored in doubles representing Barycentric Dynamical Time (seconds since
+///   2000-01-01 12:00:00). This however differs from the number of UTC seconds since that date
+///   as some leap seconds are taken into account. The conversion methods below take this into
```

Comment:
This sentence is confusing.

Reply:
I rephrased it. Do you think it's better now?